packages/timezone/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@blueprintjs/timezone",
+  "version": "1.0.0",
+  "description": "Components related to timezone selection and UI",
+  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "typings": "dist/esm/index.d.ts",
+  "style": "dist/blueprint-timezone.css",
+  "unpkg": "dist/timezone.bundle.js",
+  "scripts": {
+    "compile": "npm-run-all -p 'compile:*'",
+    "compile:esm": "tsc -p ./src",
+    "compile:cjs": "tsc -p ./src/tsconfig.cjs.json",
+    "compile:css": "sass-compile ./src",
+    "clean": "rm -rf dist/*",
+    "dev": "npm-run-all -p 'compile:esm -- --watch' 'compile:css -- --watch'",
+    "dist": "npm-run-all -s 'dist:*'",
+    "dist:bundle": "NODE_ENV=production webpack",
+    "dist:verify": "assert-package-layout",
+    "lint": "npm-run-all -p lint:stylelint lint:tslint",
+    "lint:stylelint": "stylelint -s scss 'src/**/*.scss'",
+    "lint:tslint": "tslint '{src,test}/**/*.{ts,tsx}' -c ../tslint-config -t codeFrame",
+    "lint-fix": "yarn lint:tslint --fix",
+    "test": "npm-run-all -s test:pre -p test:karma",
+    "test:pre": "tsc -p ./test",
+    "test:karma": "karma start",
+    "test:karma:debug": "karma start --single-run=false --reporters=mocha --debug",
+    "verify": "npm-run-all compile -p dist test lint"
+  },
+  "dependencies": {
+    "@blueprintjs/core": "^1.34.0",
+    "@blueprintjs/select": "^1.0.0",
+    "classnames": "^2.2",
+    "fuzzaldrin-plus": "^0.5.0",
+    "moment": "^2.14.1",
+    "moment-timezone": "^0.5.13",
+    "tslib": "^1.5.0"
+  },
+  "devDependencies": {
+    "@blueprintjs/node-build-scripts": "^0.2.0",
+    "@blueprintjs/test-commons": "^0.1.0",
```

Comment:
same here, `^0.3.0` (match semver range of whatever the package currently is in the monorepo)